app/(frontend)/contact/page.tsx

```
@@ -1,33 +1,85 @@
 
-import { BreadcrumbBar } from '@/components/products/breadcrumb-bar'
-import { FancyTitle } from '@/components/fancy-title'
-import { ContactForm } from './contact-form'
+import {BreadcrumbBar} from '@/components/products/breadcrumb-bar'
+import {FancyTitle} from '@/components/fancy-title'
+import {ContactForm} from './contact-form'
+import {LiaFacebookF, LiaGoogle, LiaInstagram, LiaTwitter} from "react-icons/lia";
+import {Footer, Media} from "@/payload-types";
+import {getCachedGlobal} from "@/utilities/getGlobals";
+import {BiCurrentLocation, BiMailSend, BiSolidPhone} from "react-icons/bi";
+import {BiCurrentLocation, BiMailSend, BiSolidPhone} from "react-icons/bi";
```

Biome (2.3.13)

[error] Shouldn't redeclare 'BiCurrentLocation'. Consider to delete it or rename it.

'BiCurrentLocation' is defined here:


(lint/suspicious/noRedeclare)